R/inputs.R

```diff
@@ -32,11 +32,16 @@ select_prgmdata <- function(prgm_dat, cnt, age_group) {
     }  } 
   
   if (!any(prgm_dat$country == cnt)) {
+
+    ## -- UPDATE HERE --
+    ## * year vecotr needs to be extended to output results to current year
+    
     prg_dat <- data.frame(country = cnt, 
-                          year = 2010:2019, 
+                          year = 2010:2020,
```

Comment:
You could set this up to run to current year using `Sys.time()`

Reply:
Yeah, I think a bit too dangerous. Needs more intensive refactoring